lessons/basics/mix.md

```diff
@@ -7,7 +7,7 @@ order: 9
 lang: en
 ---
 
-Before we can dive into the deeper waters of Elixir we first need to learn about mix. If you're familiar with Ruby mix is Bundler, RubyGems, and Rake combined. It's a crucial part of any Elixir project and in this lesson we're going to explore just a few of its great features. To see all that mix has to offer run `mix help`.
+Before we can dive into the deeper waters of Elixir we first need to learn about mix. If you're familiar with Ruby, mix is Bundler, RubyGems, and Rake combined. It's a crucial part of any Elixir project and in this lesson we're going to explore just a few of its great features. To see all that mix has to offer run `mix help`.
```

Comment:
@doomspork no, you cannot. But that's not what that sentence is conveying.
Mix is a crucial part of Elixir, as you wouldn't be able to compile witihout it, and the `mix.exs` is a crucial part of the project, because without it you wouldn't be able to compile it.
But Mix is no part the project, it's an external tool. That's what I was trying to say.